src/pudl/output/ferc1.py

```diff
@@ -1533,38 +1522,16 @@ def remove_inter_table_calc_duplication(
         return exploded
 
 
-def in_explosion_tables(
-    source_tables: list[str], in_explosion_table_names: list[str]
-) -> bool:
+def in_explosion_tables(table_name: str, in_explosion_table_names: list[str]) -> bool:
```

Comment:
we can delete this function once we delete `redefine_calculations_with_components_out_of_explosion`